docs/reference/azpol.md

```diff
@@ -0,0 +1,232 @@
+# How does Azure Policies in Enterprise-scale Landing Zone help?
+
+Azure Policy makes it possible to *codify* enterprise governance requirements. Enterprise Scale Landing Zone (referred as ESLZ hereafter) implementation extensively uses Azure Policies to make it easy to put into practice various guard rail measures necessary to meet operational and management requirements.  Most common enterprise governance requirements and how they can be set up using Azure Policies in ESLZ is discussed below.
+
+## Prevent Public IP based services
+
+Most of the Azure Platform-as-a-service (PaaS) services are created with a public IP address assigned to them. This is a good option for developers who want to quickly get started with these services. Public endpoint accelerates learning curve and is ideal when developing pilots and small-scales Proof Of Concept (PoC) implementations.
+
+However, when these pilots/PoCs make transition to production-ready enterprise applications, their use of public IP addresses is sometimes overlooked.
+
+Production workloads using public IPs without proper security measures in place can increase security risks. Malicious actors can potentially use public IP as a gateway to launch an attack. Many enterprise compliance policies do not allow use of public IP just to avoid exposure to such security risks.
+
+There is a custom ESLZ policy on denying creation of public IP address which prevents pubic IP getting created in a scope targeted by policy. Enterprises can easily prevent Virtual Machines (VMs) getting created with public IP using this policy.
+
+Similarly there is a custom ESLZ Policy Initiative (also referred as PolicySet) which  helps enterprises prevent Azure services getting created with a public IP address in the first place. Following key Azure services are prevented ever getting created with a Public IP address.
+
```

Comment:
Would recommend to review this table and make this statement "Following key Azure services are prevented ever getting created with a Public IP address." more generic.